src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,73 @@
+package nextstep.qna.domain;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+public class Answers {
+    
+    private final List<Answer> answers;
+    
+    public Answers() {
+        this(List.of());
+    }
+    
+    public Answers(Answer... answers) {
+        this(Arrays.asList(answers));
+    }
+    
+    public Answers(List<Answer> answers) {
+        this.answers = new ArrayList<>(answers);
+    }
+    
+    public void addAnswer(Answer answer) {
+        this.answers.add(answer);
+    }
+    
+    public boolean deleteAll() {
+        return this.answers.stream()
+            .map(Answer::deleteAnswer)
+            .allMatch(Answer::isDeleted);
+    }
+    
+    public boolean isAllDelete() {
```

Comment:
```suggestion
    public boolean deleteAll(NsUser loginUser) {
       ...
    }
   
```
앞의 Answer의 delete 피드백과 같음

Reply:
이유
도메인 불변식 유지 & 서비스 레이어는 다른 외부 서비스, DB 등과의 조율이 주요 목표